pbm/storage/s3/s3.go

```diff
@@ -121,9 +123,23 @@ func (s *S3) Save(name string, data io.Reader) error {
 			}
 		}
 
+		// MaxUploadParts is 1e4 so with PartSize 10Mb the max allowed file size
+		// would be ~ 97.6Gb. So if the file size is bigger we're enlarging PartSize
+		// so PartSize * MaxUploadParts could fit the file.
+		// If calculated PartSize is smaller than the default we leave the default.
+		// sizeb usually would be an uncompressed db size on disk
+		partSize := defaultPartSize
+		if sizeb > 0 {
+			ps := sizeb / s3manager.MaxUploadParts * 99 / 100 // shed 1% just to be sure we fit in even when data is not compressed
```

Comment:
Ah, wait. The SizeOnDisk property fetched in MaxDBSize() is the size of the *.wt files, but the WiredTiger storage engine has compressed their document data using s2 already.

I think we my need to do something such as: multiply by a factor if zip or no compression type is used.